### DISCUSSION TITLE
Icon Request: AR - Augmented reality

### DISCUSSION THREAD

**Discussion by Bagmaz** [FortAwesome/Font-Awesome#21216] [Icon Requests]
Please add augmented reality icon

Thank you.

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

Could you please provide a single color, 16px size suitable example?

Please take another look at our [Contributing guidelines](https://github.com/FortAwesome/Font-Awesome/blob/master/CONTRIBUTING.md#requesting-new-icons).

**Comment by Bagmaz** [comment]
http://www.arworks.com/en/wp-content/uploads/2017/03/ARIconPng.jpg

**Comment by tagliala** [comment]
A simpler example?

Icons should work at 16px

**Comment by timtjtim** [comment]
Is this still too complicated?

![screenshot from 2017-08-17 03-56-43](https://user-images.githubusercontent.com/6292691/29394093-1dfb6b78-8300-11e7-8095-687c5c6e1782.png)

16px - it's rather obscure:

![g4579](https://user-images.githubusercontent.com/6292691/29394124-4117a784-8300-11e7-94c5-ec2cc0c7ab09.png)

 Although I'd argue that lots of logos are rather unclear at 16px:

![screenshot from 2017-08-17 03-59-11](https://user-images.githubusercontent.com/6292691/29394169-873c260e-8300-11e7-8caf-893bc59fe546.png)

**Comment by vguenichon** [comment]
I think it's pretty much easier to achieve flattened. Like that :

![ar](https://user-images.githubusercontent.com/1157822/42369149-5fa7b2ae-8109-11e8-81b9-65c708dab1c6.png)

**Comment by vguenichon** [comment]
With FA5 (pro) and layering system you can achieve that.
`<span class="fa-layers fa-fw">
    <i class="far fa-expand-wide">
    <i class="fas fa-cube" data-fa-transform="shrink-8">
</span>`

**Comment by tagliala** [comment]
@vguenichon thanks for the example

**Comment by designbyadrian** [comment]
Apple automatically displays an AR icon on AR-supported links.

The icon looks like this according to the [Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines/technologies/augmented-reality/)

<img width="197" alt="image" src="https://user-images.githubusercontent.com/2064558/208127255-ad202d15-a407-4df3-af24-f849abc5e6f8.png">

Meanwhile, Google's Scene Viewer displays the button like this:

![](https://modelviewer.dev/assets/eg-default-ar-button.jpg)

The [viewfinder icons](https://fontawesome.com/search?q=viewfinder&o=r) currently in the library look like a good fit

<img width="416" alt="image" src="https://user-images.githubusercontent.com/2064558/208132199-02a79792-6115-4d91-9539-068f50e76fec.png">

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
